nmt-python-advanced-deploy-nemo-nmt-model-on-riva.ipynb

```diff
@@ -0,0 +1,404 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d447973b",
+   "metadata": {},
+   "source": [
+    "<img src=\"\" style=\"width: 90px; float: right;\">\n",
+    "\n",
+    "# How to deploy a NeMo-finetuned NMT model on Riva Speech Skills server?\n",
+    "This tutorial walks you through how to deploy a NeMo-finetuned NMT model on Riva Speech Skills server."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4e9f3a9",
+   "metadata": {},
+   "source": [
+    "## NVIDIA Riva Overview\n",
+    "\n",
+    "NVIDIA Riva is a GPU-accelerated SDK for building speech AI applications that are customized for your use case and deliver real-time performance. <br/>\n",
+    "Riva offers a rich set of speech and natural language understanding services such as:\n",
+    "\n",
+    "- Automated speech recognition (ASR)\n",
+    "- Text-to-Speech synthesis (TTS)\n",
+    "- Neural Machine Translation (NMT)\n",
+    "- A collection of natural language processing (NLP) services, such as named entity recognition (NER), punctuation, and intent classification.\n",
+    "\n",
+    "In this tutorial, we will deploy a NeMo-finetun4ed NMT model on the Riva Speech Skills server. <br> \n",
+    "Refer to the \"How to fine-tune a Riva NMT Bilingual model with Nvidia NeMo\" tutorial in [Riva NMT Tutorials](https://ngc.nvidia.com/resources/riem1phmzvud:riva:riva_nmt_ea_tutorials) to learn about finetuning Riva NMT model.<br>\n",
+    "\n",
+    "For more information about Riva, refer to the [Riva developer documentation](https://developer.nvidia.com/riva). <br> For more information about Riva NMT, refer to the [Riva NMT documentation](https://ngc.nvidia.com/resources)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "636e4272",
+   "metadata": {},
+   "source": [
+    "## NVIDIA NeMo Overview\n",
+    "\n",
+    "NVIDIA NeMo is a toolkit for building new state-of-the-art conversational AI models. NeMo has separate collections for Automatic Speech Recognition (ASR), Natural Language Processing (NLP), and Text-to-Speech (TTS) models. Each collection consists of prebuilt modules that include everything needed to train on your data. Every module can easily be customized, extended, and composed to create new conversational AI model architectures. \n",
+    "<br>  \n",
+    "For more information about NeMo, refer to the [NeMo product page](https://developer.nvidia.com/nvidia-nemo) and [documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/starthere/intro.html). The open-source NeMo repository can be found [here](https://github.com/NVIDIA/NeMo)."
```

Comment:
These links are not getting rendered correctly. 
See https://github.com/nvidia-riva/tutorials/blob/add_nmt_tutorials/nmt-python-advanced-deploy-nemo-nmt-model-on-riva.ipynb

Reply:
Same is the case in finetune notebook

Reply:
@virajkarandikar If I open it in Jupyter it is rendered without errors. Should I still try to fix it?

Reply:
May be the github rendering has issue.